debian/changelog

```diff
@@ -1,3 +1,9 @@
+privacyidea-ucs (2.19.1-1) unstable; urgency=medium
```

Comment:
This would still be 2.18.1. 
So how does the version number in mv_conffile work?

I would prefer to name this version (in changelog) 2.18.1-2.
Would the 2.18.1 in maintscript still move it - after the config file is installed.
Since the debian/privacyidea-ucs.install file installs the
``/etc/apache2/sites-available/privacyidea`` without conf...